### PR TITLE
DlgFreezecolumns

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/instat  this is ready for review

### CODE REVIEW

File: instat/dlgFreezeColumns.Designer.vb [hunk 1]
```diff
@@ -40,27 +40,28 @@ Partial Class dlgFreezeColumns
...
         '
         'lblFreezeColumns
         '
         Me.lblFreezeColumns.AutoSize = True
         Me.lblFreezeColumns.Location = New System.Drawing.Point(237, 9)
         Me.lblFreezeColumns.Name = "lblFreezeColumns"
         Me.lblFreezeColumns.Size = New System.Drawing.Size(82, 13)
-        Me.lblFreezeColumns.TabIndex = 3
+        Me.lblFreezeColumns.TabIndex = 1
         Me.lblFreezeColumns.Text = "Freeze Columns"
```
Comment (lilyclements) [inline instat/dlgFreezeColumns.Designer.vb]: We should have Freeze Columns with a colon here

Comment (dannyparsons) [inline instat/dlgFreezeColumns.Designer.vb]: Has this been done?

Comment (stevenndungu) [inline instat/dlgFreezeColumns.Designer.vb]: Doing it now

**Comment by stevenndungu** [comment]
Done

**Pull request merged**